.nvmrc

```diff
@@ -1 +1 @@
-8.15.0
+8.17.0
```

Comment:
horrifying yes, to use node 8 in 2021. However, the hugo template we based this on is still using it, so updating this to a less old version of 8 before doing anything else.